test/rauc.t

```diff
@@ -158,7 +158,10 @@ test -f ${SOFTHSM2_MOD} &&
   test_set_prereq PKCS11
 
 # Prerequisite: faketime available [FAKETIME]
+# On some platforms faketime is broken, see e.g. https://github.com/wolfcw/libfaketime/issues/418
+# Only use it if it works for date
 faketime "2018-01-01" date &&
+faketime "2018-01-15" date -R | grep "Jan 2018" &&
```

Comment:
is the change of day done on purpose? To counter negative timezones? That probably is worth a comment

Reply:
Yes, that was the idea. I find that obvious enough that I'd not add a comment, but I won't oppose strongly. Adding a comment for a single command in a &&-list is a bit ugly.

Reply:
Maybe just add a note about it in the commit message?

Reply:
Ah, actually that's wrong, 2018-01-01 *does* work even with negative timezones, it just emits
```    
Mon, 01 Jan 2018 00:00:00 -0x00
```
then. I'll use that and add a note about this being safe in the commit log then.